q2_composition/plugin_setup.py

```diff
@@ -65,6 +74,19 @@
         'transform_function': Str % Choices(_transform_functions),
         'difference_function': Str % Choices(_difference_functions)
     },
+    input_descriptions={
+        'table': 'The feature table to be used for ANCOM computation.'
+    },
+    parameter_descriptions={
+            'metadata': ('The sample metadata category to test for '
+                         'differential abundance across.'),
+            'statistical_test': ('The test to be applied to detect '
+                                 'differential abundance across groups.'),
+            'transform_function': ('The method applied to transform feature '
+                                   'values before generating volcano plots.'),
+            'difference_function': 'The method applied to compute differences '
+                                   'in feature abundances across groups '
+                                   'before generating volcano plots.'},
```

Comment:
Should it be *fold difference* rather than *fold change* (we're not measure change here, e.g., between skin and oral samples, but rather difference)?